### PR TITLE
Milestone 2: add, remove, edit functionality

### PR DESCRIPTION
**Pull request by DeVuDeveloper**
Hi,
I am done with all requirements for this project.
No linter errors.
- Add a new ToDo task with enter key. (task is added in local storage)
- Remove task with 'trash' icon which changes previous 'dots' icon on editing task text content.
  (task is removed from local storage)
- Editing text content on click, changing field background and icons.
- Ids are sorted and removing random tasks, still sorted.
Thank you!
Update: checkboxes fixed and edit now is working properly.
Hi I got feedback from code reviewer and she made mistake. This is milestone 2 and clear all and other functionality coming next. Thanks. Also I  submitted once and I see now is last try. I don't know why. If you can fix this issue pls.

### CODE REVIEW

**Review by rukundoeric — CHANGES_REQUESTED** [review]
Hi @VuDej, 

_Great job so far 👏 💯 👍. However, there are few issues we need to address before we move on. Please, check the comments below._
## Highlights
  - Follows git workflow.  💯
  - Codebase looks clean and professional. 💯 

## Required Changes ♻️
  - [ ] There are still some bugs in your app that need to be addressed. Kindly fix them. Check the Video [Here](https://www.loom.com/share/651aaad468ef4efc9f867ff72ff70ef9)
       - Check box is not working so that means `clear all completed` wouldn't work either.
       - Edit feature is not working completely, Because when your delete one of the items, all items you edited are reset back to their previous value.
       

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
  - N/A

Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/render.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    remove.appendChild(removeIcon);
+    remove.setAttribute('id', todo.index);
+    remove.style.border = 'none';
+    removeIcon.classList.add('iconR', 'fas', 'fa-ellipsis-v', 'fa-2x');
+  });
+
+  const clear = document.createElement('p');
+  tasksToDo.appendChild(clear);
+  clear.classList.add('grey');
+  clear.innerText = 'Clear all completed';
```
Comment (ZeenatLawal) [inline src/render.js:40]: - [ ] The clear all completed button doesn't clear the tasks marked as complete, kindly fix this.

File: src/constructor.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
+class ToDo {
+  constructor() {
+    this.list = localStorage.getItem('toDoTask')
+      ? JSON.parse(localStorage.getItem('toDoTask'))
+      : [];
+  }
```
Comment (ZeenatLawal) [inline src/constructor.js:6]: - [ ] The status of the task should be updated when the checkbox is clicked and the status should also be stored in the local storage. Currently, it is set to false even when the checkbox is clicked, kindly modify that.

**Review by ZeenatLawal — CHANGES_REQUESTED** [review]
Hi @VuDej, 

Great job so far!:clap: :clap:
## To Highlight 🎊 🎊 
- GitHub flow is followed. ✔️
- PR has a title and summary. ✔️

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ZeenatLawal** [comment]
To the next reviewer, kindly approve the project as all the requirements for this milestone have been met. I made a mistake and required changes that were not for this milestone, kindly confirm that. Thanks.

@VuDej I apologize for the misunderstanding.

**Comment by ShoiraTa** [comment]
### **STATUS: APPROVED  ✔️** 
------
Hi @Fransis,  

Your project is complete! Congratulations!  🎉💯🌟 



Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**